app/views/comments/new.html.erb

```diff
@@ -1,12 +1,21 @@
+
+
+    <h2> Comments</h2>
+    <% flash.each do |type, msg| %>
+      <div class="comment">
+        <%= msg %>
+      </div>
+    <% end %>
```

Comment:
- [ ] Kindly render these flash messages in the post#show view, not in the comment form